build-system/pr-check.js

```diff
@@ -29,18 +33,51 @@ function timedExecOrDie(cmd) {
   log.info('Done running', colors.cyan(cmd), '...');
 }
 
+/**
+ * Set up and execute tests for an app.
+ *
+ * TODO(rcebulko): Replace this with gulp tasks.
```

Comment:
Agree with this TODO. While you're at it, we should also apply the same logging standards that `amphtml` uses.

Reply:
Adding a TODO for logging